packages/core-data/src/resolvers.js

```diff
@@ -902,7 +902,12 @@ export const getDefaultTemplateId =
 		// Wait for the the entities config to be loaded, otherwise receiving
 		// the template as an entity will not work.
 		await resolveSelect.getEntitiesConfig( 'postType' );
-		const id = template?.wp_id || template?.id;
+		// When active_templates experiment is disabled, use string ID format (theme//slug)
```

Comment:
This comment is hard to grok because it's talking about the experiment being disabled but the condition on line 908 is when the experiment is enabled. I'd say something like:


```suggestion
		// When active_templates experiment is enabled, use  numeric wp_id if it exists, otherwise fall back to string ID format (theme//slug) as the frontend expects string IDs for templates
```

Reply:
Sounds good, thanks!